package.json

```diff
@@ -5,17 +5,19 @@
   "main": "dist/index.js",
   "type": "module",
   "scripts": {
+    "migrate:postgres": "dotenv -e test.env -- npx prisma migrate deploy",
     "compile": "rm -rf dist/ && tsc",
-    "test": "mocha test/index.ts",
-    "dev": "nodemon",
-    "start": "npm run compile && node dist/index.js"
+    "test": "dotenv -e test.env -- mocha test/index.ts",
+    "dev": "dotenv -e .env -- nodemon",
+    "start": "npm run compile && dotenv -e .env -- node dist/index.js"
```

Comment:
comentário mais a título de aprendizado e para considerar quando entrar no projeto, no onboard pode ficar do jeito que ta:
eu acho que no `test` e `dev` tudo bem forçar os arquivos .env com o dotenv, mas no `start` pode ser meio problemático se considerarmos que este é o script executado na nuvem e lá não vai ter o arquivo .env (via de regra costumamos usar secret mangers ou outras formas de injetar os envs durante a inicialização do server)